UPGRADE.md

```diff
@@ -8,6 +8,14 @@ you are using a URL-style DSN and have a username, password or database name con
 percentage sign, you need to update your DSN. If your password is, say, ``foo%foo``, it
 should be encoded as ``foo%25foo``.
 
+## MINOR BC BREAK: Native JSON type support in MySQL 5.7 platform
+
+MySQL 5.7 ships with native JSON type support which now is adopted by DBAL.
+If you are using MySQL 5.7 and have columns with ``json_array`` type defined in your database,
+utilizing DBAL's schema comparator will cause a schema diff, where the previously used native type
+``LONGTEXT`` will be changed to ``JSON``.
```

Comment:
I'm not sure this is actually true - the `Comparator` doesn't inspect the actual column type so won't pick up the change (unless/until it has some other reason to alter that column). See https://github.com/doctrine/dbal/pull/2266#issuecomment-254499121